app/__init__.py

```diff
@@ -10,6 +10,7 @@
 from users.views import users
 from queries.views import queries
 from views import main
+from models import *
```

Comment:
please avoid importing `*` . Instead prefer specific imports.

Reply:
changing that